GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -295,7 +295,8 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(FormBrowse.Command.QuickPush, Keys.Control | Keys.Shift | Keys.Up),
                     Hk(FormBrowse.Command.Stash, Keys.Control | Keys.Alt | Keys.Up),
                     Hk(FormBrowse.Command.StashPop, Keys.Control | Keys.Alt | Keys.Down),
-                    Hk(FormBrowse.Command.ToggleBranchTreePanel, Keys.Control | Keys.Alt | Keys.C)),
+                    Hk(FormBrowse.Command.ToggleBranchTreePanel, Keys.Control | Keys.Alt | Keys.C),
+                    Hk(FormBrowse.Command.JumpToArtificialCommit, Keys.Control | Keys.OemBackslash)),
```

Comment:
Hard to find how to get the default combination on my Swedish keyboard as \ requires AltGr-+ (it turned out Ctrl-< worked, so the default was convenient too). There are few available combinations...

Reply:
hotkey mentioned in the issue description

Reply:
The problem was finding how to type Keys.OemBackslash

Reply:
> The problem was finding how to type Keys.OemBackslash

I meant that I had added `Ctrl+<` to the issue description in addition.